<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>scoreboard_course_staff</class>
 <widget class="QMainWindow" name="scoreboard_course_staff">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>611</width>
      <height>421</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>#</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Student ID</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Student Name</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Total Mark</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Final Mark</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Midterm Mark</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Other Mark</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="updatescore_button">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>30</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Update</string>
    </property>
   </widget>
   <widget class="QPushButton" name="removescore_button">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>70</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Remove</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
